src/MockClassifier.Api/appsettings.json

```diff
@@ -5,5 +5,9 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "DmrServiceSettings": {
+    "DmrApiUri": "https://www.google.com",
```

Comment:
Is this okay?  Thinking out loud here...  I mean, I don't know what else you'd put...

Reply:
Please see my latest changes and let me know what you think :)

See `CONTRIBUTING.md` for more details also

Reply:
Yep - very nice.